### PR TITLE
New digidecs

### PR DESCRIPTION
**Pull request by TobiasDeBruijn**
Replaces the old digidecs with a new one

### CODE REVIEW

File: ansible/roles/digidecs/templates/digidecs.json.j2 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "server": {
```
Comment (SilasPeters) [inline ansible/roles/digidecs/templates/digidecs.json.j2:2]: You mention this hardcoded port multiple times in ansible. I suggest moving it to a variable

File: ansible/roles/digidecs/templates/digidecs.service.j2 [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# {{ ansible_managed }}
```
Comment (SilasPeters) [inline ansible/roles/digidecs/templates/digidecs.service.j2:1]: I believe the brackets are redundant

File: ansible/roles/digidecs/templates/digidecs.service.j2 [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+[Unit]
+Description=Run the digidecs backend
```
Comment (SilasPeters) [inline ansible/roles/digidecs/templates/digidecs.service.j2:4]: Does this imply that the backend starts the frontend as well?

File: ansible/roles/digidecs/templates/digidecs.service.j2 [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+OnFailure=failure-notificator@%n.service
```
Comment (SilasPeters) [inline ansible/roles/digidecs/templates/digidecs.service.j2:5]: Can't check on mobile, but did you add the matching healthcheck on healthchecks.io?

**Review by SilasPeters — CHANGES_REQUESTED** [review]

File: ansible/roles/digidecs/templates/digidecs.service.j2 (hunk 3 above)
Comment (TobiasDeBruijn) [reply ansible/roles/digidecs/templates/digidecs.service.j2:4]: No, the backend is static and served by nginx

File: ansible/roles/digidecs/templates/digidecs.service.j2 (hunk 2 above)
Comment (TobiasDeBruijn) [reply ansible/roles/digidecs/templates/digidecs.service.j2:1]: Perhaps, but we use with brackets in most places, so I prefer the consistency

File: ansible/roles/digidecs/templates/digidecs.service.j2 (hunk 4 above)
Comment (TobiasDeBruijn) [reply ansible/roles/digidecs/templates/digidecs.service.j2:5]: Not sure how to do that tbh. Couldn't find one for fallacious-rooster, on which this service is based

File: ansible/roles/digidecs/templates/digidecs.service.j2 (hunk 2 above)
Comment (SilasPeters) [reply ansible/roles/digidecs/templates/digidecs.service.j2:1]: Damn you're right.
https://jpmens.net/2020/09/29/using-ansible-managed/

File: ansible/roles/digidecs/templates/digidecs.service.j2 (hunk 4 above)
Comment (SilasPeters) [reply ansible/roles/digidecs/templates/digidecs.service.j2:5]: Appearently we have to ways of monitoring our services. This `OnFailure` is used for every systemd service, whilst only some of them use Healthchecks.io. I will have to ask when to use healthchecks.io, but this comment is resolved.

**Review by SilasPeters — APPROVED** [review]

**Pull request merged**